

<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>Results &mdash; SMS5279  documentation</title>
  

  
  
  
  

  
  <script type="text/javascript" src="../_static/js/modernizr.min.js"></script>
  
    
      <script type="text/javascript" id="documentation_options" data-url_root="../" src="../_static/documentation_options.js"></script>
        <script type="text/javascript" src="../_static/jquery.js"></script>
        <script type="text/javascript" src="../_static/underscore.js"></script>
        <script type="text/javascript" src="../_static/doctools.js"></script>
        <script type="text/javascript" src="../_static/language_data.js"></script>
    
    <script type="text/javascript" src="../_static/js/theme.js"></script>

    

  
  <link rel="stylesheet" href="../_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="../_static/pygments.css" type="text/css" />
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="Acknowledgements" href="acknowledgements.html" />
    <link rel="prev" title="Material and Methods" href="material_methods.html" /> 
</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">
    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >
          

          
            <a href="../index.html" class="icon icon-home"> SMS5279
          

          
          </a>

          
            
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <p class="caption"><span class="caption-text">Table of Contents</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="project_description.html">Project Description</a></li>
<li class="toctree-l1"><a class="reference internal" href="material_methods.html">Material and Methods</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Results</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#quality-control">Quality Control</a></li>
<li class="toctree-l2"><a class="reference internal" href="#population-structure">Population structure</a></li>
<li class="toctree-l2"><a class="reference internal" href="#imputation">Imputation</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="acknowledgements.html">Acknowledgements</a></li>
<li class="toctree-l1"><a class="reference internal" href="data_responsibility.html">Data Responsibility</a></li>
<li class="toctree-l1"><a class="reference internal" href="closing.html">Closing Procedures</a></li>
</ul>

            
          
        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../index.html">SMS5279</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="../index.html">Docs</a> &raquo;</li>
        
      <li>Results</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
            
            <a href="../_sources/content/results.rst.txt" rel="nofollow"> View page source</a>
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="results">
<span id="id1"></span><h1>Results<a class="headerlink" href="#results" title="Permalink to this headline">¶</a></h1>
<p>More info on the methods is found in the <a class="reference internal" href="material_methods.html#material-methods"><span class="std std-ref">Material and Methods</span></a> section.</p>
<div class="section" id="quality-control">
<h2>Quality Control<a class="headerlink" href="#quality-control" title="Permalink to this headline">¶</a></h2>
<p>The initial dataset consists of 10627 samples and 517724 variants. Prior to imputation, additional filtering was performed to remove lower quality SNP and samples. At <a class="reference internal" href="#filtertable"><span class="std std-numref">Table 1</span></a>, you can see an overview of the filtering steps and how many variants and samples were removed.</p>
<table border="1" class="colwidths-given docutils align-center" id="filtertable">
<caption><span class="caption-number">Table 1 </span><span class="caption-text">Overview of Filtering steps and removed SNP and Samples.</span><a class="headerlink" href="#filtertable" title="Permalink to this table">¶</a></caption>
<colgroup>
<col width="75%" />
<col width="25%" />
</colgroup>
<thead valign="bottom">
<tr class="row-odd"><th class="head">Filter</th>
<th class="head"># of removed SNP/Samples</th>
</tr>
</thead>
<tbody valign="top">
<tr class="row-even"><td>SNP call rate &lt; 0.95</td>
<td>140</td>
</tr>
<tr class="row-odd"><td>ID call rate &lt; 0.98</td>
<td>523</td>
</tr>
<tr class="row-even"><td>ID FHET outside +- 0.2</td>
<td>36</td>
</tr>
<tr class="row-odd"><td>SNP call rate &lt; 0.98</td>
<td>21981</td>
</tr>
<tr class="row-even"><td>SNP MAF &lt; 0.01</td>
<td>47945</td>
</tr>
<tr class="row-odd"><td>SNP HWE &lt; 1.0e-300</td>
<td>111</td>
</tr>
<tr class="row-even"><td>SNP sHWE &lt; 1.0e-6</td>
<td>11361</td>
</tr>
</tbody>
</table>
<p>After the filter steps, a dataset consisting of 10069 samples and 430622 variants remains as input for the imputation step.</p>
<div class="admonition note">
<p class="first admonition-title">Note</p>
<p class="last">There is a small discrepancy of around 5000 SNP between the number of filtered SNP and the remaining SNP for imputation. This is a consequence of the –exclude function in PLINK which removes listed variants AND all of their duplicates. There are quite some duplicate IDs in the original dataset called “Physical.Position_X_X” which are counted once for filtering while all of the duplicates are removed during the subsetting resulting in a slightly lower number of remaining SNP.</p>
</div>
</div>
<div class="section" id="population-structure">
<h2>Population structure<a class="headerlink" href="#population-structure" title="Permalink to this headline">¶</a></h2>
<p>To check for population structure a <a class="reference external" href="https://builtin.com/data-science/step-step-explanation-principal-component-analysis">PCA</a> was performed (see <a class="reference internal" href="#pcaplot"><span class="std std-numref">Fig. 1</span></a>). PCA helps us to identify patterns in data based on the correlation between features. In a nutshell, PCA aims to find the directions of maximum variance in high-dimensional data and projects it onto a new subspace with equal or fewer dimensions than the original one. It is often used as a visualization method where more similar data points will cluster together. In this dataset, PCA shows that - as expected - samples group more or less according to their ancestry. This will influence the HWE p-values strongly and highlights the need for the alternative HWE calculation used in the project.</p>
<div class="figure" id="pcaplot">
<img alt="../_images/PCA_plot.png" src="../_images/PCA_plot.png" />
<p class="caption"><span class="caption-number">Fig. 1 </span><span class="caption-text">PCA plot of the QCed data coloured according to self-declared ethnic group.</span></p>
</div>
</div>
<div class="section" id="imputation">
<h2>Imputation<a class="headerlink" href="#imputation" title="Permalink to this headline">¶</a></h2>
<p>Imputation using IMPUTE4 and the 1000 Genomes, phase 3 reference dataset resulted in 81.699.844 SNP in total, prior to any quality control. Overall concordance rate were calculated for a random sample of 25 “chunks” using IMPUTE2. This metric is the result of an internal cross-validation that the program performs automatically. For this analysis, IMPUTE2 masks the genotypes of one variant at a time in the study data, then imputes the masked genotypes with information from the reference data and nearby study variants. The imputed genotypes are then compared with the original genotypes to evaluate the quality of the imputation. This number should typically be around 95%; it may be lower in certain populations or regions of the genome, but if it is much lower then you may need to double-check the analysis. Overall concordances for the 25 random chunks are shown in <a class="reference internal" href="#concordance"><span class="std std-numref">Fig. 2</span></a>. For most chunks, the overall concordance rate is around 95% or higher.</p>
<div class="figure" id="concordance">
<img alt="../_images/concordance.png" src="../_images/concordance.png" />
<p class="caption"><span class="caption-number">Fig. 2 </span><span class="caption-text">Overall concordance of 25 random chunks. Indicated is the 95% marker, indicating high quality imputation.</span></p>
</div>
<p>The INFO score is an important metric indicating the uncertainty in the imputed genotype. SNP not reaching a 0.3 threshold value were removed. A representative distribution of the INFO scores is shown in <a class="reference internal" href="#infoplot"><span class="std std-numref">Fig. 3</span></a>. Similar plots for the other chromosomes can be downloaded <a class="reference download internal" download="" href="../_downloads/2594d80559ee20cc4556b556fe0ad53f/infoplots.zip"><code class="xref download docutils literal notranslate"><span class="pre">here</span></code></a>.</p>
<div class="figure" id="infoplot">
<img alt="../_images/chr1_info_hist.png" src="../_images/chr1_info_hist.png" />
<p class="caption"><span class="caption-number">Fig. 3 </span><span class="caption-text">Histogram of the INFO scores of Chromosome 1. Threshold value indicated in red.</span></p>
</div>
<p>After additional filtering of SNP with MAF &lt; 0.0001 and identical duplicates, a total of <strong>40.637.119 variants in 10.069 individuals remained</strong>.</p>
<p>The data is stored on Bianca in two folders: one containing the data split up per chromosome and one containing the whole dataset in a single file.</p>
<div class="highlight-default notranslate"><div class="highlight"><pre><span></span><span class="n">proj</span><span class="o">/</span>
<span class="o">|--</span> <span class="n">sens2020568</span>
    <span class="o">|--</span> <span class="n">nobackup</span>
        <span class="o">|--</span> <span class="n">results</span>
            <span class="o">|--</span> <span class="n">combined_data</span>
            <span class="o">|</span>   <span class="o">|--</span> <span class="n">ABCD2_imputed_03_noDup</span><span class="o">.</span><span class="p">{</span><span class="n">bim</span><span class="p">,</span><span class="n">fam</span><span class="p">,</span><span class="n">bed</span><span class="p">}</span>
            <span class="o">|--</span> <span class="n">per_chromosome</span>
                <span class="o">|--</span> <span class="n">Chr1_imputed_03_0001</span><span class="o">.</span><span class="p">{</span><span class="n">bim</span><span class="p">,</span><span class="n">fam</span><span class="p">,</span><span class="n">bed</span><span class="p">}</span>
                <span class="o">|--</span> <span class="n">Chr2_imputed_03_0001</span><span class="o">.</span><span class="p">{</span><span class="n">bim</span><span class="p">,</span><span class="n">fam</span><span class="p">,</span><span class="n">bed</span><span class="p">}</span>
                <span class="o">|--</span> <span class="o">...</span>
                <span class="o">|--</span> <span class="n">Chr22_imputed_03_0001</span><span class="o">.</span><span class="p">{</span><span class="n">bim</span><span class="p">,</span><span class="n">fam</span><span class="p">,</span><span class="n">bed</span><span class="p">}</span>
</pre></div>
</div>
</div>
</div>


           </div>
           
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="acknowledgements.html" class="btn btn-neutral float-right" title="Acknowledgements" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right"></span></a>
      
      
        <a href="material_methods.html" class="btn btn-neutral float-left" title="Material and Methods" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <p>
        &copy; Copyright 2020, Vincent van Hoef

    </p>
  </div>
  Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a <a href="https://github.com/rtfd/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  


  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>

  
  
    
   

</body>
</html>