%% Generated by Sphinx.
\def\sphinxdocclass{report}
\documentclass[letterpaper,10pt,english]{sphinxmanual}
\ifdefined\pdfpxdimen
   \let\sphinxpxdimen\pdfpxdimen\else\newdimen\sphinxpxdimen
\fi \sphinxpxdimen=.75bp\relax

\PassOptionsToPackage{warn}{textcomp}
\usepackage[utf8]{inputenc}
\ifdefined\DeclareUnicodeCharacter
% support both utf8 and utf8x syntaxes
  \ifdefined\DeclareUnicodeCharacterAsOptional
    \def\sphinxDUC#1{\DeclareUnicodeCharacter{"#1}}
  \else
    \let\sphinxDUC\DeclareUnicodeCharacter
  \fi
  \sphinxDUC{00A0}{\nobreakspace}
  \sphinxDUC{2500}{\sphinxunichar{2500}}
  \sphinxDUC{2502}{\sphinxunichar{2502}}
  \sphinxDUC{2514}{\sphinxunichar{2514}}
  \sphinxDUC{251C}{\sphinxunichar{251C}}
  \sphinxDUC{2572}{\textbackslash}
\fi
\usepackage{cmap}
\usepackage[T1]{fontenc}
\usepackage{amsmath,amssymb,amstext}
\usepackage{babel}



\usepackage{times}
\expandafter\ifx\csname T@LGR\endcsname\relax
\else
% LGR was declared as font encoding
  \substitutefont{LGR}{\rmdefault}{cmr}
  \substitutefont{LGR}{\sfdefault}{cmss}
  \substitutefont{LGR}{\ttdefault}{cmtt}
\fi
\expandafter\ifx\csname T@X2\endcsname\relax
  \expandafter\ifx\csname T@T2A\endcsname\relax
  \else
  % T2A was declared as font encoding
    \substitutefont{T2A}{\rmdefault}{cmr}
    \substitutefont{T2A}{\sfdefault}{cmss}
    \substitutefont{T2A}{\ttdefault}{cmtt}
  \fi
\else
% X2 was declared as font encoding
  \substitutefont{X2}{\rmdefault}{cmr}
  \substitutefont{X2}{\sfdefault}{cmss}
  \substitutefont{X2}{\ttdefault}{cmtt}
\fi


\usepackage[Bjarne]{fncychap}
\usepackage{sphinx}

\fvset{fontsize=\small}
\usepackage{geometry}


% Include hyperref last.
\usepackage{hyperref}
% Fix anchor placement for figures with captions.
\usepackage{hypcap}% it must be loaded after hyperref.
% Set up styles of URL: it should be placed after hyperref.
\urlstyle{same}

\addto\captionsenglish{\renewcommand{\contentsname}{Table of Contents}}

\usepackage{sphinxmessages}
\setcounter{tocdepth}{1}



\title{SMS5279}
\date{Dec 01, 2020}
\release{}
\author{Vincent van Hoef}
\newcommand{\sphinxlogo}{\vbox{}}
\renewcommand{\releasename}{}
\makeindex
\begin{document}

\pagestyle{empty}
\sphinxmaketitle
\pagestyle{plain}
\sphinxtableofcontents
\pagestyle{normal}
\phantomsection\label{\detokenize{index::doc}}
\noindent{\sphinxincludegraphics[width=0.450\linewidth]{{scilife_logo}.jpg}\hspace*{\fill}}

\noindent{\hspace*{\fill}\sphinxincludegraphics[width=0.450\linewidth]{{NBIS-logo}.png}}

\begin{DUlineblock}{0em}
\item[] 
\end{DUlineblock}




\chapter{Project Description}
\label{\detokenize{content/project_description:project-description}}\label{\detokenize{content/project_description::doc}}

\section{Support request}
\label{\detokenize{content/project_description:support-request}}
Children ages 8\sphinxhyphen{}12 years spend an average of 4\sphinxhyphen{}6 hours a day watching or using screens. However, especially in children and adolescents, consistent evidence point to the negative effects of increased screen time on several aspects such as the development of physical abilities, socio\sphinxhyphen{}emotional development, sleep, obesity, depression, anxiety, stress regulation, and mental imagery.

Based on empirical findings, Lillard et al.describe different theories that account for long\sphinxhyphen{}term media influences on executive functions. One points to the fact that media time is time away from other activities that may foster executive functions. Another highlights the interferences of attentional processes due to rapid scene changes and high levels of sensory stimulation in media.

However, previous evidence most frequently based on correlational approaches or retrospective comparisons, which are biased by numerous confounding variables and errors in estimating screen time. Therefore, causal relations between screen time and executive functions are limited.

To solve this issue, here in this project we will combine the approaches of Mendelian Randomization and Polygenic Scores based on the genotype data and screen time use of a sample of children. The sample comes from the \sphinxhref{http://abcdstudy.org}{ABCD study} and consists of N=11,875 individuals aged 9/10 years old at baseline. A wide range of measurements was collected for each individual. In addition to demographic and cognitive variables, we also obtained the whole\sphinxhyphen{}genome genotyping data.

Genotyping was performed using the Smokescreen array35, consisting of 646,247 genetic variants. Before variant imputation, quality controls (QC) on the genotyping were performed to ensure each genetic variant has been successfully called in more than 95 percent of the sample and that missingness for each individual was not higher than 20\%. After this QC 517,724 SNPs and 10,659 individuals remained.

Now, we need support to perform the SNP imputation of this QCed genotype data using the appropriate reference panel. This will allow us to later calculate polygenic scores in our sample based on the effect sizes of a large GWAS and use these for Mendelian Randomization analyses.

Our main request for NGI is to perform the imputations of the genotype data for us and then perform (if needed) a post imputation QC (to guarantee parameters such as minor allele frequency above 5\% and Hardy\sphinxhyphen{}Weinberg threshold of 10\sphinxhyphen{}6). Additionally, if there is anyone with the expertise, we could also use technical help later with the Mendelian Randomization analyses.


\section{Requested analyses}
\label{\detokenize{content/project_description:requested-analyses}}
The following analyses were agreed upon:
\begin{itemize}
\item {} 
Preprocessing of data supplied by user
\begin{itemize}
\item {} 
Additional QC for HWE, Heterozygosity rate, MAF

\item {} 
PCA to check for outliers

\item {} 
Splitting into separate chromosomes

\item {} 
Pre\sphinxhyphen{}phasing using Shapeit

\end{itemize}

\item {} 
Imputation (XY not included)
\begin{itemize}
\item {} 
Imputation using IMPUTE2 (or similar)

\item {} 
Merging of data

\item {} 
Post\sphinxhyphen{}imputation QC with info score

\item {} 
Converting to selected output format

\end{itemize}

\item {} 
Results
\begin{itemize}
\item {} 
Report including methods and parameters used, code and files

\end{itemize}

\end{itemize}


\chapter{Material and Methods}
\label{\detokenize{content/material_methods:material-and-methods}}\label{\detokenize{content/material_methods::doc}}
\begin{sphinxShadowBox}
\begin{itemize}
\item {} 
\phantomsection\label{\detokenize{content/material_methods:id1}}{\hyperref[\detokenize{content/material_methods:quality-control}]{\sphinxcrossref{Quality Control}}}

\end{itemize}
\end{sphinxShadowBox}

This is a overview of the different steps taken to clean up the data and prepare the files for imputation.


\section{Quality Control}
\label{\detokenize{content/material_methods:quality-control}}
Quality control is an important step prior to imputation.


\chapter{Results}
\label{\detokenize{content/results:results}}\label{\detokenize{content/results::doc}}
\begin{sphinxShadowBox}
\begin{itemize}
\item {} 
\phantomsection\label{\detokenize{content/results:id1}}{\hyperref[\detokenize{content/results:quality-control}]{\sphinxcrossref{Quality Control}}}

\item {} 
\phantomsection\label{\detokenize{content/results:id2}}{\hyperref[\detokenize{content/results:imputation}]{\sphinxcrossref{Imputation}}}

\end{itemize}
\end{sphinxShadowBox}


\section{Quality Control}
\label{\detokenize{content/results:quality-control}}

\section{Imputation}
\label{\detokenize{content/results:imputation}}

\chapter{Acknowledgements}
\label{\detokenize{content/acknowledgements:acknowledgements}}\label{\detokenize{content/acknowledgements::doc}}
If you are presenting the results in a paper, at a workshop or conference, we kindly ask you to acknowledge us.
\begin{itemize}
\item {} 
\sphinxstylestrong{NBIS Staff} are encouraged to be co\sphinxhyphen{}authors when this is merited in accordance to the ethical recommendations for authorship, e.g. \sphinxhref{http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html}{ICMJE recommendations}). If applicable, please include \sphinxstylestrong{Vincent van Hoef, National Bioinformatics Infrastructure Sweden, Science for Life Laboratory, Uppsala University}, as co\sphinxhyphen{}author. In other cases, NBIS would be grateful if support by us is acknowledged in publications according to this example: \sphinxhref{https://www.nbis.se/resources/support.html}{“Support by NBIS (National Bioinformatics Infrastructure Sweden) is gratefully acknowledged”}.

\item {} 
\sphinxstylestrong{UPPMAX} If your project has used HPC resources we kindly asks you to acknowledge UPPMAX and SNIC. If applicable, please add: \sphinxhref{https://www.uppmax.uu.se/support/faq/general-miscellaneous-faq/acknowledging-uppmax--snic--and-uppnex/}{“The computations were performed on resources provided by SNIC through Uppsala Multidisciplinary Center for Advanced Computational Science (UPPMAX) under Project SNIC XXXX/Y\sphinxhyphen{}ZZZ”}.

\item {} 
\sphinxstylestrong{NGI} In publications based on data from NGI Sweden, the authors must acknowledge SciLifeLab, NGI and UPPMAX: \sphinxhref{https://ngisweden.scilifelab.se/info/faq\#how-do-i-acknowledge-ngi-in-my-publication}{“The authors would like to acknowledge support from Science for Life Laboratory, the National Genomics Infrastructure, NGI, and Uppmax for providing assistance in massive parallel sequencing and computational infrastructure”}.

\end{itemize}


\chapter{Data Responsability}
\label{\detokenize{content/data_responsability:data-responsability}}\label{\detokenize{content/data_responsability::doc}}\begin{itemize}
\item {} 
\sphinxstylestrong{NBIS \& Uppnex} Unfortunately, we do not have resources to keep any files associated with the support request. We suggest that you safely store the results delivered by us. In addition, we ask that you remove the files from UPPMAX/UPPNEX after analysis is completed. The main storage at UPPNEX is optimized for high\sphinxhyphen{}speed and parallel access, which makes it expensive and not the right place for long time archiving.

\item {} 
\sphinxstylestrong{Sensitive data} Please note that special considerations may apply to the human\sphinxhyphen{}derived sensitive personal data. These should be handled according to specific laws and regulations.

\item {} 
\sphinxstylestrong{Long\sphinxhyphen{}term backup} The responsibility for data archiving lies with universities and we recommend asking your local IT for support with long\sphinxhyphen{}term data archiving. Also the newly established Data Office at SciLifeLab may be of help to discuss other options.

\end{itemize}


\chapter{Closing Procedures}
\label{\detokenize{content/closing:closing-procedures}}\label{\detokenize{content/closing::doc}}
You should soon be contacted by one of our managers, Jessica Lindvall (\sphinxhref{mailto:jessica.lindvall@nbis.se}{jessica.lindvall@nbis.se}) or Henrik Lantz (\sphinxhref{mailto:henrik.lantz@nbis.se}{henrik.lantz@nbis.se}), with a request to close down the project in our internal system and for invoicing matters. If we do not hear from you within \sphinxstylestrong{30 days} the project will be automatically closed and invoice sent. Again, we would like to remind you about data responsibility and acknowledgements, see Data Responsibility and Acknowledgments sections.

You are naturally more than welcome to come back to us with further data analysis request at any time via \sphinxhref{http://nbis.se/support/support.html}{NBIS support}.

Thank you for using NBIS and all the best for future research.

This report described the analysis of project \#5279. It contains several sections.

{\hyperref[\detokenize{content/project_description::doc}]{\sphinxcrossref{\DUrole{doc}{Project Description}}}}: A description of the projects and an overview of the requested and agreed upon analyses.

{\hyperref[\detokenize{content/material_methods::doc}]{\sphinxcrossref{\DUrole{doc}{Material and Methods}}}}: Overview of the material and methods used during the project.

{\hyperref[\detokenize{content/results::doc}]{\sphinxcrossref{\DUrole{doc}{Results}}}}: Overview of the quality control and imputation results.

\begin{sphinxadmonition}{warning}{Warning:}
Please do not forget to read the important sections on the {\hyperref[\detokenize{content/acknowledgements::doc}]{\sphinxcrossref{\DUrole{doc}{Acknowledgements}}}}, {\hyperref[\detokenize{content/data_responsability::doc}]{\sphinxcrossref{\DUrole{doc}{Data Responsability}}}} and {\hyperref[\detokenize{content/closing::doc}]{\sphinxcrossref{\DUrole{doc}{Closing Procedures}}}}.
\end{sphinxadmonition}



\renewcommand{\indexname}{Index}
\printindex
\end{document}